### PR TITLE
Backport f38 release notes

### PR DESCRIPTION
**Pull request by jkonecny12**
Backport release notes from https://github.com/rhinstaller/anaconda/pull/4716 and https://github.com/rhinstaller/anaconda/pull/4718 from Fedora 38 branch.

### CODE REVIEW

File: docs/release-notes.rst [hunk 1]
```diff
@@ -3,6 +3,138 @@ Release notes
...
+and the installation environment now contain different error messages.
+See the `#4350 <https://github.com/rhinstaller/anaconda/pull/4350>`__ for more information.
+
+The Web UI of Anaconda is now packaged in Fedora
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Anaconda team is working for some time on the new Web UI frontend for the installer and to make
+this in development Web UI more accessible to people we decided to add this as a new package to
+Fedora repositories. To be able to consume this Web UI, you need to build ISO with the Web UI
+package and add kernel boot arguments `inst.webui`. This package is not included to the existing
```
Comment (VladimirSlavik) [inline docs/release-notes.rst:134]: ...included in the...

File: docs/release-notes.rst [hunk 2]
```diff
@@ -3,6 +3,138 @@ Release notes
...
+
+Faster core dumps
+^^^^^^^^^^^^^^^^^
+
+Previously, Anaconda used a custom setup for handling tracebacks and saving core dumps. This is
+now realized by using the ``faulthandler`` Python module and the ``systemd-coredump`` service.
+As a result, the same debugging data is still available, while the installation environment
+becomes responsive significantly sooner after tracebacks. As a side effect, the logs from Anaconda
+and the installation environment now contain different error messages.
+See the `#4350 <https://github.com/rhinstaller/anaconda/pull/4350>`__ for more information.
```
Comment (VladimirSlavik) [inline docs/release-notes.rst:126]:
```
... pull request `#4350 ...
```

**Review by VladimirSlavik — COMMENTED** [review]
Maybe not worth fixing for another push :)

**Review by M4rtinK — APPROVED** [review]
Looks good to me, thanks! :)

File: docs/release-notes.rst (hunk 1 above)
Comment (jkonecny12) [reply docs/release-notes.rst]: Fixed.

File: docs/release-notes.rst (hunk 2 above)
Comment (jkonecny12) [reply docs/release-notes.rst]: Fixed.

**Pull request merged**